man/man4/zfs.4

```diff
@@ -1761,6 +1761,16 @@ Blocks that go to the special vdevs are still written indirectly, as with
 .Sy logbias Ns = Ns Sy throughput .
 This parameter is ignored if an SLOG is present.
 .
+.It Sy zfs_import_defer_txgs Ns = Ns Sy 5 Pq uint
+Number of transaction groups to wait after pool import before starting
+background work such as asynchronous block freeing
+.Pq from snapshots, clones, and deduplication
+and scrub or resilver operations.
```

Comment:
While you're at it I think we should extend this generic mechanism to cover sequential rebuilds, `vdev_rebuild_thread()`,  and background TRIM and initialize operations as well.

Reply:
IIRC they are running in a separate threads, not in sync, and this moves their start a bit later, so I hoped they should not delay the TXG commits during the import.  But I haven't looked on them very close.